src/node/internal/crypto_keys.ts

```diff
@@ -81,6 +92,12 @@ import { validateObject, validateString } from 'node-internal:validators';
 // existed first. We're, however, going to layer our KeyObject on top of
 // CryptoKey with a few augmentations.
 
+function isStringOrBuffer(val: any) {
```

Comment:
```suggestion
function isStringOrBuffer(val: any): asserts val is string | Buffer {
```

Reply:
That doesn't work here. adding the `asserts ...` is not compatible with the function returning a boolean.